src/styles/blocks/watch.scss

```diff
@@ -0,0 +1,42 @@
+@import "./../utils/mixins";
+@import "./../utils/variables";
+
+.watch {
+  left: 50%;
+  position: absolute;
```

Comment:
Don't add external styles (positioning or margins) to BEM-blocks. Use mix where necessary and move all external styles under element selector.